lib/scss_lint/linter/use_variables_for_properties.rb

```diff
@@ -0,0 +1,18 @@
+module SCSSLint
+  class Linter::UseVariablesForProperties < Linter
+    include LinterRegistry
+
+    def visit_root(_node)
+      @included_properties = Set.new(config['included_properties'])
```

Comment:
Can just call this `properties`. Since there is no concept of `excluded_properties`, the `included_` qualifier seems unnecessary.